RHDHPAI-808: Developer Lightspeed content RHDH (#1300)
redhat-developer/red-hat-developers-documentation-rhdh: modules/developer-lightspeed/con-llm-requirements.adoc

:_mod-docs-content-type: CONCEPT
[id="con-llm-requirements_{context}"]
= Large language model (LLM) requirements

> {ls-short} follows a Bring Your Own Model approach. This means that to function, {ls-short} requires access to a large language model (LLM) which you must provide. A large language model is a type of generative AI that interprets natural language and generates human-like text or audio responses. When an LLM is used as a virtual assistant, the LLM can interpret questions and provide answers in a conversational manner.

Reviewer: ```suggestion
{ls-short} follows a Bring Your Own Model approach. To function, {ls-short} requires access to a large language model (LLM) which you must provide. An LLM is a type of generative AI that interprets natural language and generates human-like text or audio responses. When used as a virtual assistant, an LLM can interpret questions and provide answers in a conversational manner.
```

Author: @jmagak , I have not fully incorporated this comment for technical accuracy and for clarity.

> LLMs are usually provided by a service or server. {ls-short} does not provide an LLM for you, so you must configure your preferred LLM provider as part of the installation process.

Reviewer: ```suggestion
LLMs are usually provided by a service or server. Since {ls-short} does not provide an LLM, so you must configure your preferred LLM provider during installation.
```

Author: I retained "for you".
